internal/proto/local/controller/api/resources/accounts/v1/account.proto

```diff
@@ -7,83 +7,90 @@ option go_package = "github.com/hashicorp/boundary/sdk/pbs/controller/api/resour
+	google.protobuf.StringValue name = 30 [(custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = {this:"name" that: "Name"}];  // @gotags: `class:"sensitive"`
 
 	// Optional user-set description for identification purposes.
-	google.protobuf.StringValue description = 40 [(custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = {this:"description" that: "Description"}];
+	google.protobuf.StringValue description = 40 [(custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = {this:"description" that: "Description"}];  // @gotags: `class:"sensitive"`
 
 	// Output only. The time this resource was created.
-	google.protobuf.Timestamp created_time = 50 [json_name="created_time"];
+	google.protobuf.Timestamp created_time = 50 [json_name="created_time"];  // @gotags: `class:"public"`
 
 	// Output only. The time this resource was last updated.
-	google.protobuf.Timestamp updated_time = 60 [json_name="updated_time"];
+	google.protobuf.Timestamp updated_time = 60 [json_name="updated_time"];  // @gotags: `class:"public"`
 
 	// Version is used in mutation requests, after the initial creation, to ensure this resource has not changed.
 	// The mutation will fail if the version does not match the latest known good version.
-	uint32 version = 70;
+	uint32 version = 70;  // @gotags: `class:"public"`
 
 	// The type of this Account.
-	string type = 80;
+	string type = 80;  // @gotags: `class:"public"`
 
 	// The ID of the Auth Method that is associated with this Account.
-	string auth_method_id = 90 [json_name="auth_method_id"];
+	string auth_method_id = 90 [json_name="auth_method_id"];  // @gotags: `class:"public"`
 
-	reserved 100;
-	reserved "attributes";
 	// The attributes that are applicable for the specific Account type.
 	oneof attrs {
-		google.protobuf.Struct default_attributes = 101 [(custom_options.v1.generate_sdk_option) = true,(custom_options.v1.subtype) = "default"];
-		PasswordAccountAttributes password_account_attributes = 102 [(custom_options.v1.generate_sdk_option) = true,(custom_options.v1.subtype) = "password"];
-		OidcAccountAttributes oidc_account_attributes = 103 [(custom_options.v1.generate_sdk_option) = true,(custom_options.v1.subtype) = "oidc"];
+		google.protobuf.Struct attributes = 100 [(custom_options.v1.generate_sdk_option) = true, (custom_options.v1.subtype) = "default"];  // @gotags: `class:"public"`
+		PasswordAccountAttributes password_account_attributes = 101 [
+			(google.api.field_visibility).restriction = "INTERNAL",
+			(custom_options.v1.generate_sdk_option) = true,
+			(custom_options.v1.subtype) = "password"
+		];  // @gotags: `class:"public"`
+		OidcAccountAttributes oidc_account_attributes = 102 [
+			(google.api.field_visibility).restriction = "INTERNAL",
+			(custom_options.v1.generate_sdk_option) = true,
+			(custom_options.v1.subtype) = "oidc"
+		];  // @gotags: `class:"public"`
```

Comment:
I added newlines between these field options, I'm happy to revert it back to a single line if we prefer.